Lib/test/test_http_cookies.py

```diff
@@ -121,6 +121,19 @@ def test_set_secure_httponly_attrs(self):
         self.assertEqual(C.output(),
             'Set-Cookie: Customer="WILE_E_COYOTE"; HttpOnly; Secure')
 
+    def test_samesite_attrs(self):
+        samesite_values = ['Strict', 'Lax']
```

Comment:
Might add tests for 'strict' and 'lax' as the values are case-insensitive from what I read.